packages/custom-decorators/test/is-enum-decorator.test.ts

```diff
@@ -0,0 +1,79 @@
+import { IsEnum } from '../src/is-enum.decorator';
+import { Validator, ValidatorOptions } from 'class-validator';
+
+function validateValues(
+  object: { testProperty: any },
+  values: any[],
+  validatorOptions?: ValidatorOptions,
+): Promise<any> {
```

Comment:
Avoid to use `any`. If there is no way to type the function properly (using an interface or generics), it is better to use `unknown`. Here, actually, it is not big deal because it is a test implementation, but it is better for readability and test maintenance.

Reply:
I see. I used the any because I got an error with `Object.entries` saying that it can't accept arguments of type unkown